meetupHelpers.js

```diff
@@ -0,0 +1,60 @@
+import { joinArrayHumanReadable } from './utils.js';
+import fetch from 'node-fetch';
+
+export async function fetchNextEventForGroup (groupId) {
+  const variables = { groupId }
+  
+  const query = `query GetUpcomingEventsForGroup ($groupId: ID) {
+    group(id: $groupId) {
+      id,
+      name,
+      upcomingEvents (input: {first: 1}) {
+        edges {
+          node {
+            dateTime,
+            timezone,
+            shortUrl,
+            tickets {
+              edges {
+                node {
+                  user {
+                    name
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }`;
+  
+  try {
+    const response = await fetch.default('https://api.meetup.com/gql', {
+      method: 'post',
+      body: JSON.stringify({query, variables}),
+      headers: {'Content-Type': 'application/json'}
+    });
+    
+    const data = await response.json();
+    
+    return data.data.group.upcomingEvents.edges[0].node;
+  } catch (e) {
+    // return undefined
```

Comment:
I think we just return null. Right now this function returns _something_ and then void. and how about a console.error() so we can debug this when there is an error?

Reply:
Nice, done!